drivers/SmartThings/sonos/src/api/sonos_ssdp_discovery.lua

```diff
@@ -300,7 +300,7 @@ function sonos_ssdp.spawn_persistent_ssdp_task()
     if info_to_send then
       if not (info_to_send.discovery_info and info_to_send.discovery_info.device) then
         log.error_with(
-          { hub_logs = true },
+          { hub_logs = false },
```

Comment:
Is there a reason we don't want to include this in hub logs?

Reply:
It can be chatty, and we've been asked by trim down the amount of logging that the LAN Edge Drivers forward to hub-core. It was set to `true` during initial development of the refactor for debugging purposes but doesn't really need to be that way anymore. With the ability to set the `broker.promoted_driver_logs` flag in LD to promote all logs for a driver, there isn't as much need for this.